meson.build

```diff
@@ -1683,6 +1684,9 @@ if get_option('libmpv')
     headers = ['libmpv/client.h', 'libmpv/render.h',
                'libmpv/render_gl.h', 'libmpv/stream_cb.h']
     install_headers(headers, subdir: 'mpv')
+
+    libmpv_test = executable('libmpv-test', 'libmpv/test.c', link_with: [libmpv])
+    test('libmpv', libmpv_test)
```

Comment:
If we now have tests in meson, would be nice to call `meson test` in all `ci/*` scripts where applicable.